packages/connect/src/protocol-grpc/trailer-status.spec.ts

```diff
@@ -17,36 +17,39 @@ import { ConnectError } from "../connect-error.js";
 import { Code } from "../code.js";
 import { Int32ValueSchema } from "@bufbuild/protobuf/wkt";
 
-describe("setTrailerStatus()", function () {
-  it("should set grpc-status when called without error", function () {
+describe("setTrailerStatus()", () => {
+  it("should set grpc-status when called without error", () => {
     const t = new Headers();
     setTrailerStatus(t, undefined);
     let count = 0;
+    // biome-ignore lint/complexity/noForEach: Headers is not iterable, and we don't have access to entries()
```

Comment:
I guess we could also add `DOM.Iterable` to `tsconfig.base.json` since they are baseline at this point. Though again, maybe not worth it for now.